ui-src/lib/models/Range90k.js

```diff
@@ -0,0 +1,71 @@
+// vim: set et sw=2:
+//
+
+import Range from './Range';
+
+/**
+ * WeakMap that keeps our private data.
```

Comment:
Why is the data kept in a WeakMap? If I'm understanding correctly, that means it can be thrown away on garbage collection. Why would we want that?

Reply:
The reason is that the JSON is now kept outside the class itself, so cannot be discovered by class users. The whole intent of this class is to hide the JSON representation.

When we're all done incorporating this change and things are working, I intend to remove the currently existing json() getter to make it truly inaccessible. We'll, a truly dedicated individual can still find it...